File: keiner_lindarte_aro/instructor_henry/mis_programas/dart/9_clase_19_de_junio/actividad.dart
import 'dart:io';

void main(){
  print("Ingrese cantidad de estudiantes:");
  int cuantos = int.parse(stdin.readLineSync()!);

  for (int i = 1; i <= cuantos; i++) {
    print("Digite su nombre acontinuacion:");
    String name = stdin.readLineSync()!;

    print("Digite su municipio acontinuacion:");
    String mun = stdin.readLineSync()!;

    print("Digite su placa acontinuacion:");
    String pla = stdin.readLineSync()!;

    var namecant = name.substring(name.length - 2);
    var muncant = mun.substring(0,3);
    var placant = pla.substring(pla.length - 3);
  
  print('Su codigo es: ' + muncant + namecant + placant);
  }


}

File: keiner_lindarte_aro/instructor_henry/mis_programas/ruby/9_clase_19_de_junio/actividad.rb
puts "Ingrese cantidad de estudiantes:"
cuantos = gets.chomp.to_i

for i in 1..cuantos

    puts "Digite su nombre acontinuacion:"
    name = gets.chomp
    puts "Digite su municipio acontinuacion:"
    mun = gets.chomp
    puts "Digite su placa acontinuacion:"
    pla = gets.chomp

    namecant = name[(name.length-2)..name.length]
    muncant = mun[0..2]
    placant = pla[(pla.length-3)..pla.length]
    puts "Su codigo es: " + muncant + namecant + placant
end